### PR TITLE
Add rich chart and blocks info

### PR DESCRIPTION
**Pull request by dnaiel0809**
Modify the contributor file.
Created a RichListChart component for displaying numbers of Bitcoin for each range of balance.
Added the queries in graphql for the Blockchair API.

### CODE REVIEW

**Comment by dnaiel0809** [comment]
@erinmhpark please kindly review my PR.

**Review by erinmhpark — APPROVED** [review]
* Great job on following the integration instruction correctly. 
* Changes don't have technical issues such as bugs.  
* The implementation is easy to understand and helpful for integration for other functionalities.

**Comment by vyasaniket** [comment]
Hey @dnaiel0809 there is a conflict in your PR can you please check once?

**Comment by dnaiel0809** [comment]
@erinmhpark @vyasaniket I merged the newest 'main' into my branch. Should not have any conflict. Please kindly review my PR again. Thanks so much! 
BTW, this PR works with branch 'dnaiel-blockchairapi' in bitcoin-grafbase repo.

**Comment by dnaiel0809** [comment]
Fix the page.tsx conflict.
Add the null data check and make sure the API response is successful.

**Review by erinmhpark — APPROVED** [review]

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
+                </div>
+                <div className={styles.blockTitle}>Time from last block</div>
+                <Timer lastBlock={lastBlock} />
                 <div>
-                    {'# blocks: ' + bitcoin.bitquery.bitcoin?.blocks?.map(b => {
-                        return b.count
-                    })}
-                    
-
                     <RichListChart></RichListChart>
```
[the remark this answers is not in the document]
Comment (dnaiel0809) [reply src/app/page.tsx]: Thanks. Fixed it.

**Review by erinmhpark — APPROVED** [review]

**Comment by Danielnewheart** [comment]
@dnaiel0809 I cannot run well maybe because of your file. Could you please fix it?
CleanShot 2024-03-19 at 13 13 59

File: src/graphql/queries.graphql [hunk 2]
```diff
@@ -9,4 +9,197 @@ query Bitcoin {
             }
         }
     }
```
Comment (rohitpanicker06) [inline src/graphql/queries.graphql:11]: Hey @dnaiel0809 
All of the queries that you created here, fail during  "npm run codegen" phase. 

Is there something else needed to run it successfully ?
Screenshot 2024-03-19 at 3 55 06 PM

**Pull request merged**